core/src/lib/StitchAppClientConfiguration.ts

```diff
@@ -16,7 +44,7 @@ export default class StitchAppClientConfiguration extends StitchClientConfigurat
     public localAppName: string;
     public localAppVersion: string;
 
-    private constructor(config?: StitchAppClientConfiguration) {
+    public constructor(config?: StitchAppClientConfiguration) {
```

Comment:
The main disadvantage of this workaround for inner classes is that the inner class constructor can no longer be private because to the outer class, the inner class declaration is at the top-level and not within the class.